ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java

```diff
@@ -184,6 +184,9 @@ public CompletableFuture<Map<String, Route>> routeFor(final RequestContext reqCt
                         ret = remote;
                     } else {
                         local.putAll(remote);
+                        for (String miss : misses) {
+                            local.putIfAbsent(miss, Route.of(miss, opts.getClusterAddress()));
```

Comment:
If a table created in a random node, will its routing relationship be updated later?

Reply:
yes, the default route not cached in routeClient